src/components/overall-statistics/ActivityPerDayChart.vue

```diff
@@ -91,6 +118,33 @@ export default class MultipleAmountPerDayChart extends Vue {
 
       case EGameMode.GM_FFA:
         return 2;
+      
+      case EGameMode.GM_4ON4_AT:
+        return 4;
+
+      case EGameMode.GM_FOOTMEN_FRENZY:
+        return 6;
```

Comment:
Should be 4

Reply:
ahh my mistake, i assumed that the footmen frenzy map was the old 3v3v3v3 = 12 players. is that not the case for this?

Reply:
It got changed to 2v2v2v2

Reply:
For w3c

Reply:
ok cool, thanks for pointing that out. made the change